index.html

```diff
@@ -1642,6 +1649,52 @@ <h3>
           </li>
         </ol>
       </section>
+      <section>
+        <h3>
+          <dfn>forget()</dfn> method
+        </h3>
+        <div class="note">
+          <p>
+            The user agent MAY decide to combine permissions across APIs, for
+            instance tracking WebHID + WebUSB device access under a unified
+            low-level device access permission. This is why this method may
+            also revoke additional (unspecified yet) permissions in the future.
+          </p>
+        </div>
+        <p>
+          The {{HIDDevice/forget()}} method steps are:
+        </p>
+        <ol>
+          <li>Let |promise:Promise| be [=a new promise=].
+          </li>
+          <li>Perform the following steps [=in parallel=]:
+            <ol>
+              <li>[=list/For each=] |device:HIDDevice| of
+              [=this=].{{HID/[[devices]]}}:
+                <ol>
+                  <li>
+                    If |device| does not share device access permissions with
+                    the HIDDevice, then continue to the next device.
+                  </li>
+                  <li>Set |device|.{{HIDDevice/[[state]]}} to `"forgetting"`.
+                  </li>
+                  <li>Invoke the user agent to revoke access to all the [=HID interfaces=]
```

Comment:
This should reject all pending promises the way that `close()` does.

Reply:
Done. Thanks!